pkgs/tools/typesetting/tex/texlive/combine.nix

```diff
@@ -31,13 +31,13 @@ let
       ++ lib.optional (lib.any pkgNeedsRuby splitBin.wrong) ruby;
   };
 
-  # TODO: replace by buitin once it exists
-  fastUnique = comparator: list: with lib;
-    let un_adj = l: if length l < 2 then l
-      else optional (head l != elemAt l 1) (head l) ++ un_adj (tail l);
-    in un_adj (lib.sort comparator list);
+  # # TODO: replace by buitin once it exists
+  # fastUnique = comparator: list: with lib;
+  #   let un_adj = l: if length l < 2 then l
+  #     else optional (head l != elemAt l 1) (head l) ++ un_adj (tail l);
+  #   in un_adj (lib.sort comparator list);
```

Comment:
Maybe just delete this? It's probably not useful to keep around.

Reply:
I meant to remove it :)